pom.xml

```diff
@@ -321,7 +321,7 @@
     <version.org.wildfly>14.0.1.Final</version.org.wildfly>
     <version.org.wildfly.arquillian>2.1.1.Final</version.org.wildfly.arquillian>
     <version.org.wildfly.core>6.0.2.Final</version.org.wildfly.core>
-    <version.org.wildfly.common>1.2.0.Final</version.org.wildfly.common>
+    <version.org.wildfly.common>1.5.0.Final-format-001</version.org.wildfly.common>
```

Comment:
Is that version string correct? `-format-001` is certainly wrong

Reply:
Fixed